### PR TITLE
:book: User stories and wireframes

### PR DESCRIPTION
**Pull request by sjd78**
- Add the UX roadmap user stories to the repo.
 
   - Add the UX wireframes to the repo.
 
   - Configure the `ci-repo.yml` action to ignore a PR that only has changes in the `docs/**` folder.
 
   - Add a lint-staged task to clean up fonts in the `*.excalidraw.svg` files.  The vscode extension works great but stores font paths that only work inside vscode.

  - Update the `vscode/.vscodeignore` file to explicitly include any file with `"../"` in the path.  Also removed duplicate entries.

Resolves: https://github.com/konveyor/kai/issues/334
Resolves: https://github.com/konveyor/editor-extensions/issues/53

### CODE REVIEW

**Review by djzager — DISMISSED** [review]
VISACK

File: docs/roadmap/stories/03-configure_analysis.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - The developer may define their own analysis configuration profiles
+
+  - Provide a set of preset analysis configurations[^1]
+    - presets would be based on what is packaged with the extension
+    - presets would be provided for the common migration targets (e.g. JEE to Quarkus, or Spring Framework to Spring Boot)
+
+  - Status indication in vscode's status bar (in addition to its runtime management function) should
+    - Indicate the active analysis configuration profile at a glance (can be similar to how git shows the current branch)
+    - Hovering the static indicator shows the full name of the analysis configuration profile
+    - Clicking the status indication...
```
Comment (rromannissen) [inline docs/roadmap/stories/03-configure_analysis.md:33]: Clicking action description is missing.

**Review by rromannissen — DISMISSED** [review]
This is brilliant. I added some comments on the user stories, as I feel like there's a bit of detachment between them and the wireframes file, but should be considered nitpicks and shouldn't prevent this from being merged.

File: docs/roadmap/stories/05-iterate_on_issue_resolution.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    - _future goal x3_: View actions, at the top level, should include an action to resolve all issues found in the project
+
+  - Status indicator for an open editor...
+    - When activating the quick pick menu, the actions should include the ability to:
+      - select a single incident in the file to resolve
+      - select a single issue in the file and resolve all of the related incidents in that file
+
+  - Similar general command pallette actions could also be made available, reusing quick pick menu actions
+
+### Reviewing and applying a resolution
```
[the remark this answers is not in the document]
Comment (sjd78) [reply docs/roadmap/stories/05-iterate_on_issue_resolution.md:53]: Yeah the detachment is a thing for sure.  The wireframes, especially for the resolution review parts, got a lot more iterations based on discussions than the stories.

Going forward, this is also the part of the project will be in the most flux over time to arrive at a smooth UX.

File: docs/roadmap/stories/03-configure_analysis.md (hunk 1 above)
Comment (sjd78) [reply docs/roadmap/stories/03-configure_analysis.md:33]: Fixed!

File: docs/roadmap/stories/03-configure_analysis.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  2. <a href="select"></a>Select an analysis configuration profile
+      - Pick one from a quick pick menu
+      - Open the profile manager
+      - Note: This action may be invoked from many other places
+
+
+## Considerations
+> [!NOTE]
+> All of the settings will drop in `settings.json` or similar file.  Therefore, all of the settings could also be configured under the standard Settings view (Ctrl+,) or directly in the json file.
```
[the remark this answers is not in the document]
Comment (sjd78) [reply docs/roadmap/stories/03-configure_analysis.md:65]: I added some extra text to try to throw an confusion to the technical designs.

**Review by djzager — APPROVED** [review]
This is awesome to have in the repo!

**Pull request merged**